data/mods/Xedra_Evolved/monsters/elementals.json

```diff
@@ -778,6 +796,227 @@
     "flags": [ "SEES", "HEARS", "SMELLS", "FAE_CREATURE", "CAN_OPEN_DOORS", "WARM" ],
     "armor": { "bash": 5, "cut": 5, "bullet": 4, "electric": 1 }
   },
+  {
+    "id": "mon_chessman_pawn",
+    "type": "MONSTER",
+    "name": { "str": "chessman: pawn" },
```

Comment:
I guess proper plural would be `chessmen: pawn`? The same for all other chessmen.